static/app/components/issueDiff/index.tsx

```diff
@@ -89,7 +132,16 @@ class IssueDiff extends Component<Props, State> {
     const event = await api.requestPromise(
       `/projects/${orgId}/${project.slug}/events/${paramEventId}/`
     );
-    return getStacktraceBody(event);
+    return event;
+  };
+
+  getTransaction = (tags: any[]) => {
+    for (let i = 0; i < tags.length; i++) {
+      if (tags[i].key === 'transaction') {
+        return tags[i].value;
+      }
+    }
+    return undefined;
```

Comment:
probably prefer a find here
```suggestion
    return tags.find(tag => tag.key === 'transaction')?.value;
```